docs/_source/community/contributing.md

```diff
@@ -57,7 +57,7 @@ Scan through our [existing issues](https://github.com/argilla-io/argilla/issues?
 
 ### Make Changes
 
-- Create [a fork](https://github.com/argilla-io/argilla/fork) of our `development` branch.
```

Comment:
We chose to name our branch `develop` because it is called "develop", so I would like to leave this naming the same.